device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C256/pg_profile_lookup.ini

```diff
@@ -0,0 +1,42 @@
+##
+## Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+## Apache-2.0
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+## http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+#  PG lossless profiles.
+# speed cable  size   xon    xoff   threshold
+  10000    5m 35840   19456  16384         0
+  25000    5m 36864   19456  17408         0
+  40000    5m 39936   19456  20480         0
+  50000    5m 41984   19456  22528         0
+ 100000    5m 59392   19456  39936         0
+ 200000    5m 65536   19456  46080         0
+ 400000    5m 99328   19456  79872         0
+ 800000    5m 128000  38912  89088         0
+  10000   40m 36864   19456  17408         0
+  25000   40m 37888   19456  18432         0
+  40000   40m 41984   19456  22528         0
+  50000   40m 44032   19456  24576         0
+ 100000   40m 63488   19456  44032         0
```

Comment:
let's set data MTU as 4k.
Using the latest excel, I got xoff 49152

Reply:
@stephenxs fixed